zilencer/tests/test_stripe.py

```diff
@@ -530,6 +530,55 @@ def check_subscription_save(subscription: stripe.Subscription, idempotency_key:
         do_activate_user(user)
         check_billing_processor_update(RealmAuditLog.USER_ACTIVATED, self.quantity + 1)
 
+    @mock.patch("stripe.Customer.save")
+    @mock.patch("stripe.Customer.retrieve", side_effect=mock_customer_with_subscription)
+    @mock.patch("stripe.Customer.create", side_effect=mock_create_customer)
+    @mock.patch("stripe.Subscription.create", side_effect=mock_create_subscription)
+    def test_card_change(self, mock_create_subscription: mock.Mock,
+                         mock_create_customer: mock.Mock, mock_retrieve_customer: mock.Mock,
+                         mock_save_customer: mock.Mock) -> None:
+        user = self.example_user("hamlet")
+        self.login(user.email)
+        response = self.client_get("/upgrade/")
+
+        self.client_post("/upgrade/", {
```

Comment:
Do we need all of this, or can we just do something like `Customer.objects.create(..)`?